### PR TITLE
Schema diagram

### PR DESCRIPTION
**Pull request by SamTush**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x] Create schema_based_on_diagram.sql of the database.
- [x] Join tables from many-to-many relationships 
- [x] Add the FK indexes.

## Description of the task to be completed ↓↓ 

- Create schema_based_on_diagram.sql of the database.

#### Thanks for reviewing our PR 😊

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+name VARCHAR(100)
+);
+
+create table invoices (
+id SERIAL PRIMARY KEY,
+total_amount DECIMAL,
+generated_at TIMESTAMP,
+payed_at TIMESTAMP,
+CONSTRAINT FK_medical_history FOREIGN KEY (medical_history_id) REFERENCES medical_histories (id)
+);
```
Comment (nahnahdev) [inline schema_based_on_diagram.sql]: - [x] Great job so far team. There are just a few changes you need to make and you'll be on your way to the next Milestone 😉 
- [x] First of all, I noticed that your code does not include the `medical_history_id` column as an integer, which is specified in the database diagram. So kindly include this as well so as to meet the project requirements 💯 

- Database Diagram
Screen Shot 2023-05-17 at 8 40 01 AM

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @SamTush and @Patfarmurs , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Followed GitFlow ✔️ 
- Great job working on the Database 🔥
- Great job for adding the Join tables from many-to-many relationships ✨💪🏻
- Professional README 

## Required Changes ♻️
- [x] Your MIT license link currently leads to an error page. I have attached a screenshot below. I will suggest that you include an MIT license file in your project and ensure that the link in your README leads to it 👍 . You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink: 
- Reference
Screen Shot 2023-05-17 at 8 58 39 AM

- [ OPTIONAL] I will suggest you add your partners information in the README as well since this is a pair-programming Milestone to also show that you contributed on this project together 😉 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @SamTush and @Patfarmurs 🙋‍♀️

Great work implementing the changes suggested by the reviewer 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Excellent branch name and PR description
-  ✅ Great work using the right Gitflow 
-  ✅ Commits are descriptive, good work
-  ✅ Database is created as expected, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Patfarmurs** [comment]
Thank you @KanzaTahreem

**Pull request merged**